src/Scan.js

```diff
@@ -321,7 +335,7 @@ class Scan extends React.Component {
       transitItem: null,
       holdItem: null,
       deliveryItem: null,
-    });
+    }, () => this.setFocusInput());
```

Comment:
Is it possible to pass `this.setFocusInput` directly instead of  `() => this.setFocusInput()`